sycl/test-e2e/NewOffloadDriver/buffer.cpp

```diff
@@ -0,0 +1,49 @@
+// REQUIRES: level_zero
+// RUN: %{build} --offload-new-driver -o %t.out
+// RUN: %{run} %t.out
+
+#include <sycl/sycl.hpp>
```

Comment:
We try to use a subset here. Please do:
```suggestion
#include <sycl/detail/core.hpp>
```